brooklyn-dist/vagrant/src/main/vagrant/servers.yaml

```diff
@@ -0,0 +1,47 @@
+---
+default_config:
+    check_newer_vagrant_box: true
+    run_os_update: true
+servers:
+  - name: brooklyn
+    box: ubuntu/vivid64
+    ram: 2048
+    cpus: 4
+    ip: 10.10.10.100
+    shell:
+      env:
+        BROOKLYN_VERSION: 0.9.0-SNAPSHOT
+      cmd:
+        - sudo sh -c 'export DEBIAN_FRONTEND=noninteractive; apt-get install --yes openjdk-8-jre-headless'
+        - curl -s -S -J -O -L "https://www.apache.org/dyn/closer.cgi?action=download&filename=brooklyn/apache-brooklyn-${BROOKLYN_VERSION}/apache-brooklyn-${BROOKLYN_VERSION}-bin.tar.gz"
```

Comment:
I'd spoken to @rdowner about this and since there isn't a consistent source for snapshots the suggestion was that we instead prompt the user to specify their own location in the server.yaml.

ie. we don't let BROOKLYN_VERSION take a snapshot value, and the user will have to update the  curl/unzip/ln steps with the snapshot they explicitly want to check out

feels a bit like sidestepping the issue tho... at least for now